src/server/components/lists/controllers/contactUsController.ts

```diff
@@ -0,0 +1,114 @@
+import type { NextFunction, Request, Response } from "express";
+import { getCSRFToken } from "server/components/cookies/helpers";
+import { countriesList } from "server/services/metadata";
+import { sendContactUsEmail } from "server/services/govuk-notify";
+import { logger } from "server/services/logger";
+import Joi from "joi";
+
+interface ContactUsFormFields {
+  country: string;
+  detail: string;
+  email: string;
+  name: string;
+  providerCompanyName?: string;
+  providerName: string;
+  serviceType: string;
+  _csrf?: string;
+}
+
+export const fieldTitles: ContactUsFormFields = {
+  country: "Which country list are you contacting us about?",
+  detail: "Provide details of why you are contacting us",
+  email: "Enter your email address",
+  name: "Enter your name",
+  providerCompanyName: "What is their company name? (Optional)",
+  providerName: "What is the service provider's name",
+  serviceType: "What service type are you contacting us about?",
+};
+
+export function getComplaintForm(req: Request, res: Response) {
+  const [errors] = req.flash("errors") as unknown as string[];
+  const errorList = errors ? (JSON.parse(errors) as Array<Record<string, string>>) : null;
+
+  if (!errorList) {
+    res.render("help/contact-us", { csrfToken: getCSRFToken(req), fieldTitles, countriesList });
+    return;
+  }
+
+  const errorsObj = errorList.reduce(
```

Comment:
could you do `errorList?.reduce?.(...)`? - that way errorsObj may be a null value and you don't have to repeat the render.